README.md

```diff
@@ -280,6 +280,10 @@ These are the available config options for making requests. Only the `url` is re
   // options are 'arraybuffer', 'blob', 'document', 'json', 'text', 'stream'
   responseType: 'json', // default
 
+  // `charset` indicates charset to use for decoding responses
+  // Note: Ignored for `responseType` of 'stream' or client-side requests
+  charset: 'utf-8', // default
```

Comment:
I'd use `responseEncoding` (adding the response prefix to make it clearer and using encoding to match the Node.js naming).